march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -34,7 +34,7 @@ using RtPublisherPtr = std::unique_ptr<realtime_tools::RealtimePublisher<T>>;
 class MarchHardwareInterface : public hardware_interface::RobotHW
 {
 public:
-  explicit MarchHardwareInterface(std::unique_ptr<march::MarchRobot> robot);
+  explicit MarchHardwareInterface(std::unique_ptr<march::MarchRobot> robot, bool use_reset_imc);
```

Comment:
```suggestion
  explicit MarchHardwareInterface(std::unique_ptr<march::MarchRobot> robot, bool reset_imc);
```
I think it is fine to just use reset_imc everywhere?